django/core/checks/security/base.py

```diff
@@ -208,3 +213,8 @@ def check_xframe_deny(app_configs, **kwargs):
 @register(Tags.security, deploy=True)
 def check_allowed_hosts(app_configs, **kwargs):
     return [] if settings.ALLOWED_HOSTS else [W020]
+
+
+@register(Tags.security, deploy=True)
```

Comment:
Since this is just a type check which should apply to all environments, not just production ones, it doesn't need the `deploy` flag